src/main/config/config_streamer.c

```diff
@@ -53,6 +53,8 @@ extern uint8_t __config_end;
 #  define FLASH_PAGE_SIZE                 ((uint32_t)0x8000)
 # elif defined(UNIT_TEST)
 #  define FLASH_PAGE_SIZE                 (0x400)
+# elif defined(SITL)
```

Comment:
Do we have some shorter name for `SIMULATOR_BUILD`?
Or just add some description in `target.h` and `README.md`?